runner.go

```diff
@@ -136,13 +175,16 @@ func Run(options ...RunOption) {
 	}
 
 	log.Println("Using config", c.ConfigPath)
```

Comment:
just checking, did we test that `taskrunner --lis` and `taskrunner --help` still work?

Reply:
I did test that locally and the corresponding test would be the one where we start the list of args with flags

Reply:
`taskrunner --help`
![Screenshot 2023-01-30 at 5 48 33 PM](https://user-images.githubusercontent.com/16676111/215622775-23676448-108c-4dbf-a19b-9c8384c3ba0b.png)

`taskrunner --list`
![Screenshot 2023-01-30 at 5 49 19 PM](https://user-images.githubusercontent.com/16676111/215622808-20dc1873-4f3c-4913-a1f7-57d3d8ac9b5b.png)